cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java

```diff
@@ -282,6 +288,33 @@ public static void main(String[] args) {
         }
     }
 
+    private static void configureLog4j() {
+        String configFilename = System.getProperty(LOG4J_CONFIGURATION_KEY);
+        String errorMessage = null;
+
+        if (StringUtils.isEmpty(configFilename)) {
+            errorMessage = "Config file name is empty.";
+        } else {
+            String normalizedPath = FilenameUtils.normalize(configFilename);
+            if (!get(normalizedPath).isAbsolute()) {
```

Comment:
@lucian-cm do you think from security perspective we should only accept absolute paths?